README.md

```diff
@@ -53,120 +53,121 @@ extraManifests:
 ```
 
 ## Customization
-The following options are supported.  See [values.yaml](/charts/atlantis/values.yaml) for more detailed documentation and examples:
-
-| Parameter                                   | Description                                                                                                                                                                                                                                                                                               | Default |
-|---------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
```

Comment:
There are a lot of changes here which makes this a bit longer to review.

Reply:
Yeah, sorry about that, I tried to make it into smaller commits, so you can follow what's happening. A lot is just fixing the formatting, which doesn't show up nicely, also ordered it, and there was a duplicate value along with a missing value.

Reply:
Side note, but ideally, this part should be auto generated with helm-docs, or similar tools, maybe it could be a good improvement idea for the future.

Reply:
Please feel free to propose a way to automate it. We'd love that because that reduces the maintenance and need for code review in the markdown.

Reply:
Created an issue for it: https://github.com/runatlantis/helm-charts/issues/230